ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -637,38 +848,57 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-minimal-permission
-- as: gcp-ipi-oidc-rt-fips-regen-cert-f28
-  cron: 9 16 13 * *
+- as: gcp-ipi-ovn-ipsec-arm-f28
+  cron: 41 12 11 * *
   steps:
     cluster_profile: gcp-qe
     env:
-      EXTRACT_MANIFEST_INCLUDED: "true"
-      FIPS_ENABLED: "true"
-      RT_ENABLED: "true"
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      ENABLE_OTA_TEST: OCP-21588
+      OCP_ARCH: arm64
     test:
-    - chain: openshift-e2e-test-qe-regenerate-cluster-cert
     - chain: openshift-upgrade-qe-test
-    workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
-- as: gcp-ipi-ovn-ipsec-arm-f14
-  cron: 26 18 14,28 * *
+    workflow: cucushift-installer-rehearse-gcp-ipi-ovn-ipsec
+- as: gcp-ipi-ovn-ipsec-capi-arm-mixarch-f28
```

Comment:
I want to highlight another another rule further for the mixarch ci jobs. If we have mixarch ci jobs, we can skip the single-arch install profile.

E.g: `gcp-ipi-ovn-ipsec-capi-arm-mixarch` can cover `gcp-ipi-ovn-ipsec-arm`, so I think we should drop `gcp-ipi-ovn-ipsec-arm`

Reply:
yeah, agree. But sounds like it should be maintained in e2e jobs. for example, when someone adds gcp-ipi-ovn-ipsec-capi-arm-mixarch, he should drop gcp-ipi-ovn-ipsec-arm from e2e jobs. If so, when we add upgrade jobs, we will only add upgrade job for gcp-ipi-ovn-ipsec-capi-arm-mixarch

Reply:
```
$ grep -r "as: gcp-ipi-ovn-ipsec" *release-4.16*
...
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:- as: gcp-ipi-ovn-ipsec-amd-mixarch-f28-destructive
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:- as: gcp-ipi-ovn-ipsec-capi-arm-mixarch-f14
```
Actually I did not see any problem in e2e jobs, `capi` just using different capi to launch cluster instead of terraform, it is a feature from installer, does not impact upgrade ci jobs.

Reply:
Removed gcp-ipi-ovn-ipsec

Reply:
yeah, the pr was not synced with the latest e2e job so we see some improper jobs.